cool-core/src/test/java/com/nus/cool/functionality/csv.properties

```diff
@@ -0,0 +1,14 @@
+#//Nomral case
+cube_0 = health
```

Comment:
We put static data under `resources` directory

Reply:
i can move under resources